### ISSUE TITLE
Integration with Marble from Chisel mod?

### ISSUE THREAD

**Issue by ventolin** [Shukaro/Artifice#6]
Hello! This mod is shaping up rather nicely! I'm messing around with installing a bunch of aesthetic mods, including Artifice and Chisel (http://www.minecraftforum.net/topic/1749374-). One thing I've noticed straight away is that both mods have marble worldgen, and different decorative marble blocks. A few of them look almost identical, and it would be cool if there was some way to combine them (perhaps through the oreDictionary? I don't know if that would work here).

In other words, a single source of marble in the world that worked with both mods, and a single kind of Marble Brick and Chiseled Marble (which Chisel also has).

It might not be possible since Chisel has a tool (the chisel) that you can use to left-click on a block in the world and change its texture, but I wanted to at least bring it to your attention.
